### PR TITLE
Unify listing application with status/event for sparkctl

### PR DESCRIPTION
**Pull request by mrow4a**
Result is: 
```
+----------+-------+---------------+---------------+
|   NAME   | STATE | SUBMITTED AGE | COMPLETED AGE |
+----------+-------+---------------+---------------+
| spark-pi | NEW   | N.A.          | N.A.          |
+----------+-------+---------------+---------------+
```

or 

```
+----------+-----------+---------------+---------------+
|   NAME   |   STATE   | SUBMITTED AGE | COMPLETED AGE |
+----------+-----------+---------------+---------------+
| spark-pi | COMPLETED | 4m            | 2m            |
+----------+-----------+---------------+---------------+
```

### CODE REVIEW

File: sparkctl/cmd/list.go [hunk 1]
```diff
@@ -50,9 +51,17 @@ func doList(crdClientset crdclientset.Interface) error {
 		return err
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "State", "Submitted age", "Completed age"})
```
Comment (liyinan926) [inline sparkctl/cmd/list.go]: `Submission Age` and `Completion Age` sound better?

**Comment by mrow4a** [comment]
@liyinan926 do you know what can be the error in travis? it happens only with go 1.9

**Comment by liyinan926** [comment]
This error happens sometime and seems transient.

File: sparkctl/cmd/list.go (hunk 1 above)
Comment (liyinan926) [reply sparkctl/cmd/list.go]: @mrow4a can you update this PR to make the suggested fixes? Will merge after they get fixed.

**Comment by mrow4a** [comment]
@liyinan926 done

**Pull request merged**